runtime/missingmember_test.go

```diff
@@ -836,15 +836,15 @@ pub contract GarmentNFT: NonFungibleToken {
        // read Garment data.
        //
        pub fun borrowNFT(id: UInt64): &NonFungibleToken.NFT {
-           return &self.ownedNFTs[id] as &NonFungibleToken.NFT
+           return (&self.ownedNFTs[id] as &NonFungibleToken.NFT?)!
        }
 
        // Parameters: id: The ID of the NFT to get the reference for
        //
        // Returns: A reference to the NFT
        pub fun borrowGarment(id: UInt64): &GarmentNFT.NFT? {
            if self.ownedNFTs[id] != nil {
-               let ref = &self.ownedNFTs[id] as auth &NonFungibleToken.NFT
+               let ref = (&self.ownedNFTs[id] as auth &NonFungibleToken.NFT?)!
                return ref as! &GarmentNFT.NFT
```

Comment:
Even though this makes the code valid, and the code itself is just a test case for something else and was provided by the community, we should maybe still update this code so it gracefully handles the case and to avoid developers seeing this and copying it as-is.

Concretely, in the cases where the functions return an optional (e.g. here, but not in the function above), maybe instead use the nil-coalescing operator `??` to return `nil` if the dictionary lookup fails, instead of aborting the execution:

```suggestion
               let ref = &self.ownedNFTs[id] as auth &NonFungibleToken.NFT?
               return ref ?? ref as! &GarmentNFT.NFT
```